testsuite/features/support/env.rb

```diff
@@ -103,6 +103,16 @@ def enable_assertions
       debug_server_on_realtime_failure
     end
   end
+  page.instance_variable_set(:@touched, false)
+end
+
+# Reset the page if it's running the first scenario of a feature
+$first_scenario = true
+Before do
```

Comment:
too bad we don't have hooks at feature level, it would avoid running this for every scenario :-(

Reply:
yup, let's move to java, there they implemented it :trollface: